.prettierignore

```diff
@@ -1,15 +1,18 @@
-node_modules/**
+node_modules/**/*
 lavamoat/**/policy.json
-dist/**
-builds/**
-test-*/**
-coverage/
-jest-coverage/
-storybook-build/
+dist/**/*
+builds/**/*
+test-*/**/*
 app/vendor/**
-.nyc_output/**
-test/e2e/send-eth-with-private-key-test/**
+.nyc_output/**/*
+test/e2e/send-eth-with-private-key-test/**/*
 *.scss
 development/chromereload.js
 development/ts-migration-dashboard/filesToConvert.json
-public
+development/ts-migration-dashboard/build/**
+public/**/*
+development/charts/**
+node_modules/**/*
+storybook-build/**/*
+jest-coverage/**/*
+coverage/**/*
```

Comment:
I've merged this file with the ignore list in our eslint config, and then updated the eslint config to use this file for its ignore list